### ISSUE TITLE
1 politician left to play... or not

### ISSUE THREAD

**Issue by wfdd** [everypolitician/gender-balance#336]
![screen shot 2016-06-18 at 02 31 27](https://cloud.githubusercontent.com/assets/8489551/16167375/d55e6a52-34fc-11e6-869b-a7e1ba1a8969.png)
![screen shot 2016-06-18 at 02 31 30](https://cloud.githubusercontent.com/assets/8489551/16167378/dc5e359e-34fc-11e6-8c8d-370d4ea3652e.png)

**Comment by wfdd** [comment]
I'm getting the same thing with a few other countries.  I've a hunch it's related to skipping.

**Comment by tmtmtmtm** [comment]
Yep — I'm getting that on a few countries too. Will investigate.

**Comment by chrismytton** [comment]
Pull request that should fix this here: https://github.com/everypolitician/gender-balance/pull/366
